Gemfile

```diff
@@ -48,7 +48,7 @@ gem 'laa-criminal-applications-datastore-api-client',
     require: 'datastore_api'
 
 gem 'laa-criminal-legal-aid-schemas',
-    github: 'ministryofjustice/laa-criminal-legal-aid-schemas', tag: 'v1.0.40'
+    github: 'ministryofjustice/laa-criminal-legal-aid-schemas', ref: 'bde24bc59345dc433c8730379074c8990dc965e0'
```

Comment:
Note: currently making use of [this draft pr](https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas/pull/100) to add savings to schema